src/lv_demo_printer/lv_demo_printer.c

```diff
@@ -994,8 +994,9 @@ static lv_anim_value_t anim_path_triangle(const lv_anim_path_t * path, const lv_
 {
     /*Calculate the current step*/
     uint32_t step;
+    lv_anim_value_t ret = 0;
```

Comment:
I'm confused about how this would fix the warning, or how the warning is present to begin with. As far as I can tell, all three branches of the `if-else` chains already return a value.

Reply:
The IAR compiler gives the following warning,
Warning[Pe111]: statement is unreachable lv_demo_printer.c 1021

Reply:
I did not answer you question correctly, there is no need to initialize ```ret``` to  0